src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -576,6 +563,24 @@ def create_compile_context(self, target, target_workdir):
         sources=sources,
       ))
 
+  # TODO: This overrides the same method in jvm_compile.py, and that method should be refactored to
```

Comment:
It's possible that rather than overriding this method, you should consider modifying `self._set_directory_digests_for_valid_target_classpath_directories` instead, to support populating digests for all contexts...?